flows-advanced/etl/export-airbyte-config-and-write-to-s3-bucket-using-blocks.py

```diff
@@ -0,0 +1,123 @@
+"""
+Uses Prefect 2.3.1
+
+Assumes you have:
+- a Secret block named `airbyte-hostname-dev` (this is just 'localhost')
+- a Secret block named `airbyte-hostname-prod`
+- an AwsCredentials block named `prefect-aws-credentials` with the following layout:
+{
+    "aws_access_key_id": "my-access-key-id",
+    "aws_secret_access_key": "my-secret-access-key"
+}
+- an S3Bucket block named `my-airbyte-config-s3bucket` with the following layout:
+{
+    "bucket_name"="my-bucket",
+    "aws_credentials"=AwsCredentials block,
+}
+
+If you are unable to create these blocks in the UI because S3Bucket or
+AwsCredentials block types do not exist,
+run
+`prefect block register -m prefect_aws.credentials`
+`prefect block register -m prefect_aws.s3`
+to register them in your workspace.
```

Comment:
Are these comments needed anymore? The S3 and credentials block are all a part of the S3 block right?

Reply:
No, they live in prefect-aws

Reply:
AwsCredentials and S3Bucket do, anyway. Not the S3 storage block.